modules/exploits/linux/local/apport_abrt_chroot_priv_esc.md

```diff
@@ -0,0 +1,69 @@
+## Description
+
+  This module attempts to gain root privileges on Ubuntu and Fedora systems by invoking the default coredump handler inside a namespace ("container").
+
+
```

Comment:
Can you add a h3 here about how to make a fedora 20 vulnerable?  The instructions in your last comment basically, i'd like to capture that in the docs for configuring the host correctly and not having to look back a year or two from now and try to remember